### ISSUE TITLE
Allow plugin variables

### ISSUE THREAD

**Issue by ponceta** [opengisch/QField#5887]
### Describe the issue
Right now, https://github.com/opengisch/qfield-geomapfish-locator publishes a template for `geomapfish-locator` plugin with two variables :

```
      "service_url": "https://geomapfish-demo-2-8.camptocamp.com/search",
      "service_crs": "EPSG:2056"
```

I'm sure this could be the case of many plugins in the future so it would be great to enable the user to set these plugin variables directly from QFIELD interface.

**Comment by nirvn** [comment]
@ponceta , can we close this her and add it in the plugin repository as with an [ enhancement ] label? Glad to see some feedback :)

**Comment by ponceta** [comment]
Yes of course! Thank you @nirvn for the great work. I managed to publish a release for our geomapfish service here :

https://github.com/VilleDePully/qfield-geomapfish-locator/releases/tag/1.1

But I think it would be wise that not every user has to deploy such coding assets to connect its geomapfish services.
